support/src/main/java/org/jenkinsci/plugins/workflow/support/storage/SimpleXStreamFlowNodeStorage.java

```diff
@@ -150,6 +151,7 @@ private Tag(@Nonnull FlowNode node, @Nonnull List<Action> actions) {
     private static final Field FlowNode$parentIds;
 
     static {
+        XSTREAM.addCompatibilityAlias("com.cloudbees.workflow.cps.checkpoint.NotExecutedNodeAction", NotExecutedNodeAction.class);
```

Comment:
Yup, nothing wrong. Rendering issue on my side yesterday I guess. (the code was completely on the left as if it was not indented).
Or I was tired... Or both :-).
Anyway, it's fine! :+1: